content/en/docs/Installation/_index.md

```diff
@@ -355,11 +355,16 @@ The following flags are used to control the behavior of Kyverno and must be set
 The following flags can also be used to control the advanced behavior of Kyverno and must be set on the main `kyverno` container in the form of arguments.
 
 1. `-v`: Sets the verbosity mode of Kyverno log output. Takes an integer from 1 to 6 with 6 being the most verbose.
-2. `--background-scan`: The interval (like 30s, 15m, 12h) for background processing resulting in policy report entries. Default is set to 1h.
-3. `--webhookTimeout`: Sets the timeout for a webhook response from Kyverno. Value is in seconds. This is deprecated in Kyverno 1.5.0.
-4. `--gen-workers`: the number of workers for processing generate policies concurrently. Default is set to 10.
-5. `--generateSuccessEvents`: specifies whether (true/false) to generate success events. Default is set to "false".
-6. `--autoUpdateWebhooks`: auto-configuration of the webhooks based on installed policies. Default is set to "true".
+2. `profile`: setting this flag to 'true' will enable profiling.
```

Comment:
@anushkamittal20 I don't see "webhookTimeout" in this PR.

https://github.com/kyverno/kyverno/blob/e5e849acfebd0461ae147d066821fdf86c620fc7/cmd/kyverno/main.go#L73-L98

Reply:
Can you also add the `-v` flag, please? Even though this isn't Kyverno specific, users need to know this is a flag that can be passed to the main container to turn up verbose logging.

Reply:
`webhookTimeout ` has been deprecated, should use `spec.webhookTimeoutSeconds` instead.

Reply:
Should I add something like 
```
--spec.webhookTimeoutSeconds: Sets the timeout for a webhook response from Kyverno. Value is in seconds. This is deprecated in Kyverno 1.5.0.
``` 
@chipzoller @realshuting

Reply:
It's a field in a policy or webhook config `spec`, right, @realshuting ?